### PR TITLE
New version: MuhammadFareezIqmal.IIUMSchedule version 1.4.3.0

### PR DESCRIPTION
**Pull request by iqfareez**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/146128)

### CODE REVIEW

File: manifests/m/MuhammadFareezIqmal/IIUMSchedule/1.4.3.0/MuhammadFareezIqmal.IIUMSchedule.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: MuhammadFareezIqmal.IIUMSchedule
+PackageVersion: 1.4.3.0
```
Comment (OfficialEsco) [inline manifests/m/MuhammadFareezIqmal/IIUMSchedule/1.4.3.0/MuhammadFareezIqmal.IIUMSchedule.installer.yaml:5]:
```suggestion
PackageVersion: 1.4.1.0
```

File: manifests/m/MuhammadFareezIqmal/IIUMSchedule/1.4.3.0/MuhammadFareezIqmal.IIUMSchedule.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: MuhammadFareezIqmal.IIUMSchedule
+PackageVersion: 1.4.3.0
```
Comment (OfficialEsco) [inline manifests/m/MuhammadFareezIqmal/IIUMSchedule/1.4.3.0/MuhammadFareezIqmal.IIUMSchedule.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.4.1.0
```

File: manifests/m/MuhammadFareezIqmal/IIUMSchedule/1.4.3.0/MuhammadFareezIqmal.IIUMSchedule.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: MuhammadFareezIqmal.IIUMSchedule
+PackageVersion: 1.4.3.0
```
Comment (OfficialEsco) [inline manifests/m/MuhammadFareezIqmal/IIUMSchedule/1.4.3.0/MuhammadFareezIqmal.IIUMSchedule.yaml:5]:
```suggestion
PackageVersion: 1.4.1.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Sadly the DisplayVersion comes back lower than the installer...
```
Enter a pull request number: 146128

Manifest validation succeeded.
Found IIUM Schedule [MuhammadFareezIqmal.IIUMSchedule] Version 1.4.3.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/iiumschedule/iium_schedule/releases/download/1.4.2%2B41/iiumschedulesetup.exe
  ██████████████████████████████  11.2 MB / 11.2 MB
Successfully verified installer hash
Starting package install...
Successfully installed

Name          : IIUM Schedule
Publisher     : Muhammad Fareez Iqmal
Version       : 1.4.1.0
ProductCode   : {570E3DBF-1549-4F5D-8D92-888BC1EE5A6C}_is1
Registry Path : Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Uninstall\{570E3DBF-1549-4F5D-8D92-888BC1EE5A6C}_is1
Uninstall     : winget uninstall --id "{570E3DBF-1549-4F5D-8D92-888BC1EE5A6C}_is1"

Press any key to continue . . .
```

**Comment by iqfareez** [comment]
Will check soon

**Comment by iqfareez** [comment]
Updated with the correct installer link

**Comment by stephengillie** [comment]
Verify: 
Apps and features Settings page for IIUM Schedule 1.4.3.0

> Sadly the DisplayVersion comes back lower than the installer...
> 
> ```
> Enter a pull request number: 146128
> 
> Manifest validation succeeded.
> Found IIUM Schedule [MuhammadFareezIqmal.IIUMSchedule] Version 1.4.3.0
> This application is licensed to you by its owner.
> Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
> Downloading https://github.com/iiumschedule/iium_schedule/releases/download/1.4.2%2B41/iiumschedulesetup.exe
>   ██████████████████████████████  11.2 MB / 11.2 MB
> Successfully verified installer hash
> Starting package install...
> Successfully installed
> 
> Name          : IIUM Schedule
> Publisher     : Muhammad Fareez Iqmal
> Version       : 1.4.1.0
> ProductCode   : {570E3DBF-1549-4F5D-8D92-888BC1EE5A6C}_is1
> Registry Path : Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Uninstall\{570E3DBF-1549-4F5D-8D92-888BC1EE5A6C}_is1
> Uninstall     : winget uninstall --id "{570E3DBF-1549-4F5D-8D92-888BC1EE5A6C}_is1"
> 
> Press any key to continue . . .
> ```

Are these pulling data from different Registry keys?

File: manifests/m/MuhammadFareezIqmal/IIUMSchedule/1.4.3.0/MuhammadFareezIqmal.IIUMSchedule.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: MuhammadFareezIqmal.IIUMSchedule
+PackageVersion: 1.4.3.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
```
Comment (stephengillie) [inline manifests/m/MuhammadFareezIqmal/IIUMSchedule/1.4.3.0/MuhammadFareezIqmal.IIUMSchedule.installer.yaml:7]:
```suggestion
InstallerType: inno
Dependencies:
  PackageDependencies:
     - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```
image

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-03-26 09:16:45.446 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!00007FFC2E1BC11F: (caller: 00007FFC2DF48D6F) LogHr(1) tid(580) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFC2E0C2B08: (caller: 00007FFC2E16BC20) Exception(1) tid(580) 8A15005E ] 
> 2024-03-26 09:16:45.446 [REPO] Failed to open available source: msstore
> 2024-03-26 09:16:53.218 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!00007FFC11DBC11F: (caller: 00007FFC11B48D6F) LogHr(1) tid(404) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFC11CC2B08: (caller: 00007FFC11D6BC20) Exception(1) tid(404) 8A15005E ] 
> 2024-03-26 09:16:53.218 [REPO] Failed to open available source: msstore
> 2024-03-26 09:16:54.205 [CLI ] No suitable installer found for manifest Microsoft.VCRedist.2015+.x64 with version 14.40.33617.1

(Automated response - build 876.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**